src/actions/player/playerTrendsActions.js

```diff
@@ -63,9 +68,22 @@ export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch)
           start_time: match.start_time,
           win,
         });
-
         return dataList;
       }, []);
+
+      // Apply mean-smoothing by taking differences of the previously computed integral
+      for (let i = 0; i < trends.length; i += 1) {
+        if (i < trends.length - chunkSize) {
+          trends[i].value = (trends[i + chunkSize].value - trends[i].value) / chunkSize;
+        } else {
```

Comment:
Done. For the subheader text, what would be preferable? I see there are various internationalization files, do I update the statement in all of them. I think we can go with "Moving average by 20 games over last 480 games"